lib/service/cfg.go

```diff
@@ -347,12 +347,27 @@ type ProxyConfig struct {
 	Kube KubeProxyConfig
 }
 
+func (c ProxyConfig) KubeAddr() (string, error) {
+	if !c.Kube.Enabled {
+		return "", trace.NotFound("kubernetes support not enabled on this proxy")
+	}
+	if len(c.Kube.PublicAddrs) > 0 {
+		return c.Kube.PublicAddrs[0].String(), nil
+	}
+	host := "<replace this with proxy hostname>"
```

Comment:
I think we usually use `<proxyhost>` as a placeholder in other situations where this might occur.